release/tag/goreleaser.yaml

```diff
@@ -19,15 +19,26 @@ builds:
     goos:
       - darwin
       - linux
-      - windows
     goarch:
```

Comment:
add a comment at the top of this file explaining how's it consumed and link to goreleaser

Reply:
Added a small paragraph